ebuild-format.rst

```diff
@@ -87,3 +86,19 @@ incidentally depend on multi-valued variable having a single value
 goes against the principle of least surprise.  Furthermore, it makes
 it hard to copy-paste part of the URI e.g. to investigate the directory
 index.
+
+
+.. index:: keywords; one line
+
+KEYWORDS must be defined on a single line
+-----------------------------------------
+:Source: QA
+:Reported: no
+
+The ``KEYWORDS`` variable must be defined on a single line, with literal
+content (no variable references, line wrapping, appending, etc.).
```

Comment:
please add wording to make it definitely clear that there can only be a single definition in the whole in order to avoid the -9999 trap.

Reply:
Added 'at most once'. That should raise no confusion that it's fine to skip it entirely.